charts/opensearch/templates/statefulset.yaml

```diff
@@ -4,22 +4,14 @@ kind: StatefulSet
 metadata:
   name: {{ template "opensearch.uname" . }}
   labels:
-    heritage: {{ .Release.Service | quote }}
-    release: {{ .Release.Name | quote }}
-    chart: "{{ .Chart.Name }}"
-    app: "{{ template "opensearch.uname" . }}"
-    {{- range $key, $value := .Values.labels }}
-    {{ $key }}: {{ $value | quote }}
-    {{- end }}
+    {{- include "opensearch.labels" . | nindent 4 }}
```

Comment:
See previous comment, if I remember correctly. Statefulsets will not allow updates to the spec other than the template, replicas and updateStrat.

So, if someone upgraded this would break.

Reply:
The `spec` starts a few lines below - I think metadata is not immutable.

Reply:
@smlx Let me triple check this right now.